### PR TITLE
Fix for fanfiction.net

### PR DESCRIPTION
**Pull request by lerndmina**
The dynamic mode inverted a lot of the page content which causes it to turn an ugly yellow. I've fixed this by using the neutral background var to allow the user to customise what actually is shown.

### CODE REVIEW

File: src/config/dynamic-theme-fixes.config [hunk 1]
```diff
@@ -23885,6 +23885,54 @@ body {
 
 ================================
 
+www.fanfiction.net
```
Comment (alexanderby) [inline src/config/dynamic-theme-fixes.config:23888]: Could you please leave just `fanfiction.net`? If so please preserve the alphabetical order.

**Review by alexanderby — CHANGES_REQUESTED** [review]

**Comment by lerndmina** [comment]
Accidentally marked as resolved. I'll fix this soon.

File: src/config/dynamic-theme-fixes.config (hunk 1 above)
Comment (lerndmina) [reply src/config/dynamic-theme-fixes.config]: Sure, I've done so now.

**Review by bershanskiy — APPROVED** [review]
The www was removed and alphabetic order was fixed.

**Pull request merged**